templates/taxi/index.html

```diff
@@ -0,0 +1,9 @@
+{% extends "base.html" %}
+
+{% block content %}
+    <ul>
+        <li>Number of cars: {{ num_cars }}</li>
+        <li>Number of drivers: {{ num_drivers }}</li>
+        <li>Number of manufacturers: {{ num_manufacturers }}</li>
+    </ul>
+{% endblock %}
```

Comment:
It's better to always end files with a newline character. You can read why this is the case [here](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline), in the top answer.